source/DataElement/AMDSFlatArray.cpp

```diff
@@ -399,6 +399,67 @@ bool AMDSFlatArray::setValue(int index, double  value)
 	return true;
 }
 
+const QVector<double> AMDSFlatArray::asConstVectorDouble() const
+{
+	QVector<double> targetVector;
+	switch(dataType_){
+	case AMDSDataTypeDefinitions::Signed8:
+		foreach(qint8 value, vectorQint8_) {
+			targetVector.append((double)value);
```

Comment:
Noted. My bust. I should have caught that. 
@XuetaoLiu-CLS , can you make an issue and fix these up?